### ISSUE TITLE
508-defect-1 ❗ Launchblocker [COGNITION, MOBILE DESIGN]: Sidenav MUST not be cut off

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8958]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When viewed in narrow viewport or on mobile, the navigation **must** not be cut off by other page components.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a mobile user, I want to view the contents of the sidenav without any content being obscured.

## Environment

* Device: mobile
* Operating System: all
* Browser: all
* Server destination: https://staging.va.gov/pittsburgh-health-care/health-services/

## Steps to Recreate

1. Enter https://staging.va.gov/pittsburgh-health-care/health-services/ in browser on mobile device
1. Verify that the sidenav only displays the hamburger icon in both portrait and landscape view 
1. Verify that the text "In this section" is not visible

## Screenshots or Trace Logs

![IMG_7446](https://user-images.githubusercontent.com/57469/81583906-06f46b80-9380-11ea-8da3-84a102846a92.PNG)

**Comment by mmiddaugh** [comment]
should be ready for your review @jenstrickland

**Comment by jenstrickland** [comment]
Validated.
